src/extra/AutoUIApp/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Navbar } from './Navbar';
+import { Sidebar } from './Sidebar';
+import { Router, Switch, Route } from 'react-router-dom';
+import { Content } from './Content';
+import { createGlobalStyle } from 'styled-components';
+import { Provider } from '../..';
+import { history } from '../AutoUI/utils';
+import { Flex } from '../../components/Flex';
+import { Box } from '../../components/Box';
+import { OpenApiJson } from './openApiJson';
+
+const GlobalStyle = createGlobalStyle`
+	html,
+	body {
+		height: 100%;
```

Comment:
The indents seem bigger in this file than in the others

Reply:
Or maybe the indents of the first few files are just smaller? 🤔 I see this file (and subsequent files) are indenting with tabs, while earlier ones indent with spaces ...

Reply:
Seems like it may have been my github view